packages/@ama-sdk/core/src/plugins/custom-info/custom-info.reply.ts

```diff
@@ -18,15 +18,18 @@ export class CustomInfoReply<G = { [key: string]: any }, V extends { [key: strin
    * Checks if the reply has a customInfo node
    * @param reply
    */
-  // eslint-disable-next-line no-use-before-define
-  public hasCustomInfo<T extends { [key: string]: any; customInfo?: K } | undefined = { [key: string]: any; customInfo: any },
-    K = { [key: string]: any }>(reply: T): reply is T & { customInfo: K & G } {
+  public hasCustomInfo<
+    // eslint-disable-next-line no-use-before-define -- order has no importance for typings
```

Comment:
The order may have an importance when it comes to generic type default values
For example, if we want the default type to be `customInfo: T` instead of any, we cannot do that:
```
<
  T extends { [key: string]: any; customInfo?: K } | undefined = { [key: string]: any; customInfo: K},
  K = { [key: string]: any }
>
```
we could however do it the other way around
```
<
  K = { [key: string]: any },
  T extends { [key: string]: any; customInfo?: K } | undefined = { [key: string]: any; customInfo: K}
>
```
changing the order would be a breaking change tough
I'm ok to disable the rule here, just not sure about the justification

Reply:
Could you propose a new sentence for the justification then please ?